app/javascript/components/applications/application_form/preferences/utils.js

```diff
@@ -1,14 +1,12 @@
-import { join, get, map, concat, pickBy } from 'lodash'
+import { get, map, concat, pickBy } from 'lodash'
+import { domainDateOfBirthToApi } from '~/components/mappers/utils'
```

Comment:
i noticed that the `domainDateOfBirthToApi` function uses the 'YYYY-MM-DD' date format string that is also used in several other places in the app. what do you think @akegan about setting this as a constant?